core/kazoo_documents/src/kzd_accounts.erl.src

```diff
@@ -19,6 +19,7 @@
 -export([do_not_disturb/1, do_not_disturb/2, set_do_not_disturb/2]).
 -export([do_not_disturb_enabled/1, do_not_disturb_enabled/2, set_do_not_disturb_enabled/2]).
 -export([enabled/1, enabled/2, set_enabled/2]).
+-export([flags/1, flags/2, set_flags/2]).
```

Comment:
Please migrate all the new accessor functions into the `.erl`

Reply:
done :)